.releaserc

```diff
@@ -0,0 +1,55 @@
+{
+  "branches": [ 
+    "release/*", { 
+      "name": "main",
+      "prerelease": "beta"
+      }
+    ],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    [
+      "@google/semantic-release-replace-plugin",
+      {
+        "replacements": [
+          {
+            "files": ["gradle.properties"],
+            "from": "version=.*",
+            "to": "version=${nextRelease.version}",
+            "results": [
+              {
+                "file": "gradle.properties",
+                "hasChanged": true,
+                "numMatches": 1,
+                "numReplacements": 1
+              }
+            ],
+            "countMatches": true
+          }
+        ]
+      }
+    ],
+    [
+      "@semantic-release/git",
+      {
+        "assets": ["gradle.properties"]
```

Comment:
Should remove `[skip ci]` from the message, it shouldn't trigger the action again since it's from the github-actions bot. Even if the commit triggers the action again, semantic-release will not produce any new artifacts.

This is required for merging the `trunk` into the `release`
```json
{
"assets": ["gradle.properties"]
"message": "chore(release): ${nextRelease.version} \n\n${nextRelease.notes}"
}
```